cspell.json

```diff
@@ -124,7 +124,8 @@
     "wordprocessingml",
     "stbr",
     "ukwest",
-    "tfsproduction"
+    "tfsproduction",
+    "clamav"
```

Comment:
This list is mostly in alphabetical order except the last few entries. Can we put this in the relevant place?